pom.xml

```diff
@@ -55,8 +55,8 @@
 		<!-- Driver Dependencies -->
 		<org.osgi.core.version>6.0.0</org.osgi.core.version>
 		<azure-security-keyvault-keys.version>4.7.3</azure-security-keyvault-keys.version>
-		<azure-identity.version>1.12.2</azure-identity.version>
-		<msal.version>1.15.1</msal.version>
+		<azure-identity.version>1.15.0</azure-identity.version>
+		<msal.version>1.17.2</msal.version>
```

Comment:
why 1.17.2? latest is 1.18.0

Reply:
Dependency version conflicts, latest azure-identity 1.15.0 depends on msal 1.17.2. If I use 1.18.0 it breaks things.